<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>subWindowClass</class>
 <widget class="QMainWindow" name="subWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1540</width>
    <height>870</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>subWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1541</width>
      <height>871</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(234, 234, 234);
border:none;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QToolButton" name="toolButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>400</width>
       <height>300</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
.QToolButton{
border-style:none;
border:1px solid #B6B6B6;
color:#57595B;
padding:5px;
min-height:15px;
border-radius:5px;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #E4E4E4,stop:1 #E4E4E4);
border-width: 1px;border-style: solid;border-color:rgb(59, 176, 176);
image: url(:/icon_3axusv3ezyt/update.png);
;
}

.QToolButton:hover{
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #9e9e9e,stop:1 #9e9e9e);
}

.QToolButton:pressed{
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #E4E4E4,stop:1 #E4E4E4);
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_2">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>70</y>
       <width>400</width>
       <height>300</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
.QToolButton{
border-style:none;
border:1px solid #B6B6B6;
color:#57595B;
padding:5px;
min-height:15px;
border-radius:5px;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #E4E4E4,stop:1 #E4E4E4);
border-width: 1px;border-style: solid;border-color:rgb(59, 176, 176);
image: url(:/icon_3axusv3ezyt/update.png);
}

.QToolButton:hover{
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #9e9e9e,stop:1 #9e9e9e);
}

.QToolButton:pressed{
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #E4E4E4,stop:1 #E4E4E4);
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>1160</x>
       <y>150</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>配准模型</string>
     </property>
    </widget>
    <widget class="QComboBox" name="lineMatch">
     <property name="geometry">
      <rect>
       <x>1250</x>
       <y>140</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox::down-arrow,QDateEdit[calendarPopup=&quot;true&quot;]::down-arrow,QTimeEdit[calendarPopup=&quot;true&quot;]::down-arrow,QDateTimeEdit[calendarPopup=&quot;true&quot;]::down-arrow{
image:url(:/qss/flatwhite/add_bottom.png);
width:10px;
height:10px;
right:2px;
font: 12pt &quot;楷体&quot;;
border-color: rgb(59, 176, 176);
}
QLineEdit,QTextEdit,QPlainTextEdit,QSpinBox,QDoubleSpinBox,QComboBox,QDateEdit,QTimeEdit,QDateTimeEdit{
border:1px solid #B6B6B6;
border-radius:3px;
padding:2px;
background:none;
selection-background-color:#FFFFFF;
selection-color:#00BB9E;
	border-color: rgb(59, 176, 176);
}

QComboBox::drop-down,QDateEdit::drop-down,QTimeEdit::drop-down,QDateTimeEdit::drop-down{
subcontrol-origin:padding;
subcontrol-position:top right;
width:15px;
border-left-width:0px;
border-left-style:solid;
border-top-right-radius:3px;
border-bottom-right-radius:3px;
border-left-color:#B6B6B6;
font: 12pt &quot;楷体&quot;;
}

QComboBox::drop-down:on{
top:1px;
font: 12pt &quot;楷体&quot;;
}</string>
     </property>
     <item>
      <property name="text">
       <string>Lightglue</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>1160</x>
       <y>220</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>融合模型</string>
     </property>
    </widget>
    <widget class="QComboBox" name="lineFuse">
     <property name="geometry">
      <rect>
       <x>1250</x>
       <y>210</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox::down-arrow,QDateEdit[calendarPopup=&quot;true&quot;]::down-arrow,QTimeEdit[calendarPopup=&quot;true&quot;]::down-arrow,QDateTimeEdit[calendarPopup=&quot;true&quot;]::down-arrow{
image:url(:/qss/flatwhite/add_bottom.png);
width:10px;
height:10px;
right:2px;
font: 12pt &quot;楷体&quot;;
border-color: rgb(59, 176, 176);
}
QLineEdit,QTextEdit,QPlainTextEdit,QSpinBox,QDoubleSpinBox,QComboBox,QDateEdit,QTimeEdit,QDateTimeEdit{
border:1px solid #B6B6B6;
border-radius:3px;
padding:2px;
background:none;
selection-background-color:#00BB9E;
selection-color:#FFFFFF;
	border-color: rgb(59, 176, 176);
}

QComboBox::drop-down,QDateEdit::drop-down,QTimeEdit::drop-down,QDateTimeEdit::drop-down{
subcontrol-origin:padding;
subcontrol-position:top right;
width:15px;
border-left-width:0px;
border-left-style:solid;
border-top-right-radius:3px;
border-bottom-right-radius:3px;
border-left-color:#B6B6B6;
font: 12pt &quot;楷体&quot;;
}

QComboBox::drop-down:on{
top:1px;
font: 12pt &quot;楷体&quot;;
}</string>
     </property>
     <item>
      <property name="text">
       <string>CDDFuse</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DenseFuse</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="buttonMatch">
     <property name="geometry">
      <rect>
       <x>1170</x>
       <y>300</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
.QPushButton,.QToolButton{
border-style:none;
border:1px solid #B6B6B6;
color:#57595B;
padding:5px;
min-height:15px;
border-radius:5px;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #E4E4E4,stop:1 #E4E4E4);
border-width: 1px;border-style: solid;border-color:rgb(59, 176, 176);
font: 10pt &quot;楷体&quot;;
}

.QPushButton:hover,.QToolButton:hover{
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #F6F6F6,stop:1 #F6F6F6);
}

.QPushButton:pressed,.QToolButton:pressed{
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #E4E4E4,stop:1 #E4E4E4);
}

.QToolButton::menu-indicator{
image:None;
}

QToolButton#btnMenu,QPushButton#btnMenu_Min,QPushButton#btnMenu_Max,QPushButton#btnMenu_Close{
border-radius:3px;
color:#57595B;
padding:3px;
margin:0px;
background:none;
border-style:none;
}

QToolButton#btnMenu:hover,QPushButton#btnMenu_Min:hover,QPushButton#btnMenu_Max:hover{
color:#FFFFFF;
margin:1px 1px 2px 1px;
background-color:rgba(51,127,209,230);
}

QPushButton#btnMenu_Close:hover{
color:#FFFFFF;
margin:1px 1px 2px 1px;
background-color:rgba(238,0,0,128);
}
</string>
     </property>
     <property name="text">
      <string>多模态配准</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonFuse">
     <property name="geometry">
      <rect>
       <x>1330</x>
       <y>300</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
.QPushButton,.QToolButton{
border-style:none;
border:1px solid #B6B6B6;
color:#57595B;
padding:5px;
min-height:15px;
border-radius:5px;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #E4E4E4,stop:1 #E4E4E4);
border-width: 1px;border-style: solid;border-color:rgb(59, 176, 176);
font: 10pt &quot;楷体&quot;;
}

.QPushButton:hover,.QToolButton:hover{
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #F6F6F6,stop:1 #F6F6F6);
}

.QPushButton:pressed,.QToolButton:pressed{
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #E4E4E4,stop:1 #E4E4E4);
}

.QToolButton::menu-indicator{
image:None;
}

QToolButton#btnMenu,QPushButton#btnMenu_Min,QPushButton#btnMenu_Max,QPushButton#btnMenu_Close{
border-radius:3px;
color:#57595B;
padding:3px;
margin:0px;
background:none;
border-style:none;
}

QToolButton#btnMenu:hover,QPushButton#btnMenu_Min:hover,QPushButton#btnMenu_Max:hover{
color:#FFFFFF;
margin:1px 1px 2px 1px;
background-color:rgba(51,127,209,230);
}

QPushButton#btnMenu_Close:hover{
color:#FFFFFF;
margin:1px 1px 2px 1px;
background-color:rgba(238,0,0,128);
}
</string>
     </property>
     <property name="text">
      <string>多模态融合</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>1170</x>
       <y>370</y>
       <width>291</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar{text-align:center;background-color:#DDDDDD;border: 1px solid #ff0000;}
QProgressBar::chunk{background-color:#05B8CC;}&quot;);

QProgressBar{text-align:center;background-color:#DDDDDD;border: 0px solid #DDDDDD;border-radius:5px;}
QProgressBar::chunk{background-color:#05B8CC; border-radius: 5px;}
 
QProgressBar{text-align:center;background-color:#DDDDDD;border: 0px solid #DDDDDD;border-radius:5px;}
QProgressBar::chunk{background-color:#05B8CC;border-radius:5px; width:8px;margin:0.5px;}&quot;);
 
QProgressBar{height:22px; text-align:center; font-size:14px; color:white; border-radius:11px; background:#1D5573;}
QProgressBar::chunk{border-radius:11px;background:qlineargradient(spread:pad,x1:0,y1:0,x2:1,y2:0,stop:0 #99ffff,stop:1 #9900ff);}&quot;);
</string>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>390</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 18pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>光学模态</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>390</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 18pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>红外模态</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_result">
     <property name="geometry">
      <rect>
       <x>1010</x>
       <y>410</y>
       <width>521</width>
       <height>391</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width: 1px;border-style: solid;border-color:rgb(59, 176, 176);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>450</y>
       <width>400</width>
       <height>300</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
.QToolButton{
border-style:none;
border:1px solid #B6B6B6;
color:#57595B;
padding:5px;
min-height:15px;
border-radius:5px;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #E4E4E4,stop:1 #E4E4E4);
border-width: 1px;border-style: solid;border-color:rgb(59, 176, 176);
image: url(:/icon_3axusv3ezyt/update.png);
}

.QToolButton:hover{
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #9e9e9e,stop:1 #9e9e9e);
}

.QToolButton:pressed{
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #E4E4E4,stop:1 #E4E4E4);
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_4">
     <property name="geometry">
      <rect>
       <x>579</x>
       <y>450</y>
       <width>401</width>
       <height>300</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
.QToolButton{
border-style:none;
border:1px solid #B6B6B6;
color:#57595B;
padding:5px;
min-height:15px;
border-radius:5px;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #E4E4E4,stop:1 #E4E4E4);
border-width: 1px;border-style: solid;border-color:rgb(59, 176, 176);
image: url(:/icon_3axusv3ezyt/update.png);
}

.QToolButton:hover{
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #9e9e9e,stop:1 #9e9e9e);
}

.QToolButton:pressed{
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #E4E4E4,stop:1 #E4E4E4);
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>760</y>
       <width>165</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 18pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>多光谱模态1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>760</y>
       <width>165</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 18pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>多光谱模态2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>1210</x>
       <y>810</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 18pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>融合结果</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="earth_map_demo.qrc"/>
 </resources>
 <connections/>
</ui>
